### ISSUE TITLE
Visualization result of Sintel

### ISSUE THREAD

**Issue by Ecalpal** [cattaneod/MaskFlownet-Pytorch#6]
Hi, cattaneod!
Thank you for implementing the code on pytorch!

I think there are some mistakes in this re-implement.

This is the ground truth from sintel training set.(ambush_6 frame_0005)
![frame_0005](https://user-images.githubusercontent.com/39435901/91457801-5d49d380-e8b7-11ea-8a4a-d57f768a6d01.png)

This is the result which I ran with the original implementation (mxnet MaskFlowNetS).
![frame_0005](https://user-images.githubusercontent.com/39435901/91457844-69ce2c00-e8b7-11ea-8168-57e06edfa764.jpg)

This is the result which you show on the readme page.
![image](https://user-images.githubusercontent.com/39435901/91458051-aac64080-e8b7-11ea-9397-6d3354fe07f5.png)

This is the result (label) which visualizated from your dataloader.
![image](https://user-images.githubusercontent.com/39435901/91458304-ee20af00-e8b7-11ea-8d00-4b1b2f569d00.png)

This is the ground truth from sintel training set.(bamboo_1 frame_0001)
![frame_0001](https://user-images.githubusercontent.com/39435901/91458563-3e980c80-e8b8-11ea-8926-028653f0ddbd.png)

Obviously, there are some problems. Can you fix it?

**Comment by cattaneod** [comment]
Hi,
the differences arise from the fact that the original implementation of MaskFlownet predict the flipped flow.
Therefore, the label are also flipped, as you can see [here](https://github.com/cattaneod/MaskFlownet-Pytorch/blob/master/predict.py#L71).

To get the final results, just flip the output of the network.
